### ISSUE TITLE
Initial shbdeviceidentifier setup does not create table entries for influx user

### ISSUE THREAD

**Issue by TB-DevAcc** [jo-jstrm/SmartHomeBuddy#72]
**Describe the bug**
When cloning the repo and setting up the project for development, `shbdeviceidentifier start` does create the sqlite tables `users` and `influxdb` but does not populate them with the necessary entries. 

**To Reproduce**
1. Remove all installed files from the SHB project
2. Clone Repo and setup the dev environment (python, influxdb)
3. start the identifier, no error is displayed
4. Check the SQLite db file, which should be empty now

**Expected behavior**
SQLite db should contain at least one user

**Desktop (please complete the following information):**
 - OS: Ubuntu

**Comment by TB-DevAcc** [comment]
This should probably already fix the issue:
Changing this line 

https://github.com/jo-jstrm/SmartHomeBuddy/blob/ac2a7cbe82cee8581b9284479b65db33257afc64/device-identifier/shbdeviceidentifier/db.py#L186

to 

``` python
elif influxdb_admin.token != None:
```

**Comment by TB-DevAcc** [comment]
> This should probably already fix the issue: Changing this line
> 
> https://github.com/jo-jstrm/SmartHomeBuddy/blob/ac2a7cbe82cee8581b9284479b65db33257afc64/device-identifier/shbdeviceidentifier/db.py#L186
> 
> to
> 
> ```python
> elif influxdb_admin.token != None:
> ```

Doesn't really fix it, the token stays empty. The question is how to set the token when the setup was run but the database does not contain the token?

**Comment by jo-jstrm** [comment]
To the best of my knowledge, there is no way to *programmatically* acquire the admin token from an already set-up InfluxDB instance. This means that you can *a)* reset InfluxDB (delete `~/.influxdbv2` and rerun SHB, which will then set InfluxDB up and add the admin token to `influxdb`) or *b)* look up the admin token in the InfluxDB GUI and add it to the DB manually.

In the future, we should probably implement a way to update the token (maybe also username/password) via SHBs GUI...

**Comment by TB-DevAcc** [comment]
Yes it should not, but it did. I have not setup Influxdb myself, meaning that shbdeviceidentifier must have set up the database but failed somehow. If this happens for a user, requiring him to delete the influxdb directory manually does not seem to be the best solution. 

Finding the influxdb data directory and remembering it might be important for the uninstaller anyway, so deleting it programmatically would then be a viable option.

**Comment by jo-jstrm** [comment]
Oh, ok.

So the logic would be to do something like (pseudo code)
```python
if is_influxdb_set_up and is_token_in_sqlite_empty:
    reset_influxdb()
    reset_user_in_sqlite()
    _run_influxdb_setup() # Already exists in db.py:132
```
